mvt/ios/modules/mixed/sms_attachments.py

```diff
@@ -55,6 +55,11 @@ def serialize(self, record: dict) -> Union[dict, list]:
 
     def check_indicators(self) -> None:
         for attachment in self.results:
+            # Check for known malicious filenames.
+            if self.indicators.check_file_path(attachment["filename"]):
+                print("Found malicious filename", attachment["filename"])
```

Comment:
Should this use a logger instance?

Reply:
Doh, yes thanks! Not sure how I missed the`print()` when committing.  The `indicators.check_file_path()` already logs the detection. We can just append the attachment to the detections without adding an additional log line.